### ISSUE TITLE
[Feature Request] PSV/VMP Export/Signing (Reverse Engineered in 2019!)

### ISSUE THREAD

**Issue by wmxp** [ShendoXT/memcardrex#12]
I know it's been years this you worked on this treasure to the community (which we appreciate you dearly for), - if you're up for a bit of a challenge, the crypto on PSV and VMP was fully defeated last year by a team with tools made open source:
https://github.com/dots-tb/vita-mcr2vmp

https://github.com/dots-tb/ps3-psvresigner

https://github.com/PMStanley/PSV-Exporter

The PS community would love to see MemcardRex make a great splash again on fan news sites to kick off 2020! 

Cheers Shendo. :)

**Comment by bucanero** [comment]
if you need to convert + resign .mcs PS1 saves to .PSV (PS3), you can try my psv converter tool:
https://github.com/bucanero/psv-save-converter

it also supports many additional PS2/ps1 formats and exports the result to PSV

**Comment by wmxp** [comment]
> if you need to convert + resign .mcs PS1 saves to .PSV (PS3), you can try my psv converter tool:
> https://github.com/bucanero/psv-save-converter
> 
> it also supports many additional PS2/ps1 formats and exports the result to PSV

Hey man, I've been playing with your PS3 software toys a bit this year. Fantastic work - having on console signing bypassing the need for PS3BSD is particularly welcome. Thanks for the heads up. 👍 

Your projects and work are much appreciated. :)

**Comment by bucanero** [comment]
> Hey man, I've been playing with your PS3 software toys a bit this year. Fantastic work - having on console signing bypassing the need for PS3BSD is particularly welcome. Thanks for the heads up. 👍
> 
> Your projects and work are much appreciated. :)

thanks for your comments! 👍  Keep an eye for new versions (or just keep the auto-update option active to get notified) 😄 

btw, with the latest version of [Apollo Save Tool](https://github.com/bucanero/apollo-ps3/releases/tag/v1.4.2) you can actually import, convert, and resign PS1/PS2 saves directly on the PS3. (e.g. import a ps2 `.max` save to `.PSV`)

**Comment by wmxp** [comment]
> btw, with the latest version of [Apollo Save Tool](https://github.com/bucanero/apollo-ps3/releases/tag/v1.4.2) you can actually import, convert, and resign PS1/PS2 saves directly on the PS3. (e.g. import a ps2 `.max` save to `.PSV`)

I saw that update in the changelog. I'm actually and old mod from the now defunct psx-scene.com, and one of the contributors to the "PSV conversion services" for the public - me being an early adopter with a CECHA and Sony memory card adapter. 

For YEARS I tried to get traction in the community on getting the PSV format cracked. I always figured flatz was our best hope, but he didn't seem to be interested. So yeah, it's nice to finally be able to put all this to bed, but I still think having easy to use GUI Windows based tools for the job is the way to go - albiet anyone still messing with PS3 this late in the game,  that hasn't gone the CFW/HEN route seems a minority these days.
